template.go

```diff
@@ -371,6 +372,9 @@ func (scope VarMap) SetWriter(name string, v SafeWriter) VarMap {
 
 // Execute executes the template in the w Writer
 func (t *Template) Execute(w io.Writer, variables VarMap, data interface{}) error {
+	if w == nil {
```

Comment:
This default we should skip, we need to fail when the user pass an nil writer. We can even panic with a descriptive error message.